README.md

```diff
@@ -35,4 +35,22 @@ vagrant@ubuntu-impish:~/CVE-2021-4034$
 
 ## About Polkit pkexec for Linux
 
-Polkit (formerly PolicyKit) is a component for controlling system-wide privileges in Unix-like operating systems. It provides an organized way for non-privileged processes to communicate with privileged processes. It is also possible to use polkit to execute commands with elevated privileges using the command pkexec followed by the command intended to be executed (with root permission).
+Polkit (formerly PolicyKit) is a component for controlling system-wide privileges in Unix-like operating systems. It provides an organized way for non-privileged processes to communicate with privileged processes. It is also possible to use polkit to execute commands with elevated privileges using the command pkexec followed by the command intended to be executed (with root permission
```

Comment:
What's with this change?

Reply:
Oops, mistype when pasting the next section, sorry.